legal/index.md

```diff
@@ -49,7 +49,7 @@ Any testimonials, opinions, advice, product or service offers, or other informat
 
 ### Materials Posted by Users
 
-By posting or submitting any material to this website, including on the forum, you will be (a) granting The Monero Project a world-wide fully paid-up non-exclusive license to allow your materials to be edited and displayed by The Monero Project on its website, and accessed by Users, for the purposes and on the terms in these TOU, (b) agreeing to indemnify and hold harmless The Monero Project and its Users from any claim or demand made by any third party due to or arising out of your materials, and (c) representing that you own or otherwise have the right to post the materials, that the materials are accurate, that use of the materials on the The Monero Project website does not violate any laws, and that the materials will not cause injury to any person. The Monero Project shall have the right to use your name in connection with your posted materials.
+By posting or submitting any material to this website, including on the forum, you will be (a) granting The Monero Project a world-wide fully paid-up non-exclusive license to allow your materials to be edited and displayed by The Monero Project on its website, and accessed by Users, for the purposes and on the terms in these TOU, (b) agreeing to indemnify and hold harmless The Monero Project and its Users from any claim or demand made by any third party due to or arising out of your materials, and (c) representing that you own or otherwise have the right to post the materials, that the materials are accurate, that use of the materials on the Monero Project website does not violate any laws, and that the materials will not cause injury to any person. The Monero Project shall have the right to use your name in connection with your posted materials.
```

Comment:
```suggestion
By posting or submitting any material to this website, including on the forum, you will be (a) granting The Monero Project a world-wide fully paid-up non-exclusive license to allow your materials to be edited and displayed by The Monero Project on its website, and accessed by Users, for the purposes and on the terms in these TOU, (b) agreeing to indemnify and hold harmless The Monero Project and its Users from any claim or demand made by any third party due to or arising out of your materials, and (c) representing that you own or otherwise have the right to post the materials, that the materials are accurate, that use of the materials on The Monero Project website does not violate any laws, and that the materials will not cause injury to any person. The Monero Project shall have the right to use your name in connection with your posted materials.
```